redesign/src/styles/_vars.scss

```diff
@@ -0,0 +1,7 @@
+$red: #981E32;
+$black: #212121;
+$lightblack:#3A3A3A;
```

Comment:
Consider updating your variable names to `$blacklight` and `$reddark` or some variation (`$black-light`, `$red-dark`). In general, I find it ends up being helpful to have all the variations of a color named similarly vs having a dozen colors that all start with `$lightsomething`.